sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -289,7 +289,15 @@ public void update(@NonNull Observable observable, @NonNull Object arg) {
      * session everytime the app enters the foreground.
      */
     public void startSession() {
-        sessionTracker.startNewSession(new Date(), user, false);
+        sessionTracker.startSession(false);
+    }
+
+    public final void stopSession() {
```

Comment:
There is a static method the Bugsnag object to call `startSession` can we add ones for stop and resume as well?